lib/db.js

```diff
@@ -573,7 +573,6 @@ var createCollection = function(self, name, options, callback) {
  * @param {boolean} [options.serializeFunctions=false] Serialize functions on any object.
  * @param {boolean} [options.strict=false] Returns an error if the collection does not exist
  * @param {boolean} [options.capped=false] Create a capped collection.
- * @param {boolean} [options.autoIndexId=true] Create an index on the _id field of the document, True by default on MongoDB 2.2 or higher off for version < 2.2.
```

Comment:
We actually say that for < 2.2 it is `false`, and for >= 2.4 it is `true`, so I think we need to remove documentation about 2.2, but still leave docs in here as well as a mention that it is deprecated.